utilities/score/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:16.04
+FROM python:3
```

Comment:
Why is this needed?

Reply:
It reports an error for python2, like
`File "entrypoint.py", line 63, in eval_TER
    file_tgt.write(('%s\t(%d-)\n' % (line.rstrip(), i)).encode('utf-8'))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 0: ordinal not in range(128)
`
I tried but seams we have to use **io.open()** with 'utf-8' encoding 
which is a little bit complex.